toolchain/check/import_cpp.cpp

```diff
@@ -1274,22 +1271,41 @@ static auto ImportDeclAndDependencies(Context& context, SemIR::LocId loc_id,
   return inst_id;
 }
 
+// Maps `clang::AccessSpecifier` to `SemIR::AccessKind`.
+static auto MapAccess(clang::AccessSpecifier access_specifier)
+    -> SemIR::AccessKind {
+  switch (access_specifier) {
+    case clang::AS_public:
+    case clang::AS_none:
```

Comment:
Does this `AS_none` come up in practice? In what cases (default/implicit access levels?)? Are those cases tested? (might be good to have test cases showing a struct and a class without explicit access modifiers on their member get handled the right way?)